.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  push:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5
+
+      - name: Install pip
+        run: pip install -r requirements/pip.txt
+
+      - name: Install Dependencies
+        run: pip install setuptools wheel
+
+      - name: Build package
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish to PyPi
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: edx
```

Comment:
See my question about the username in the similar django-user-tasks PR.

Reply:
`${{ secrets.PYPI_UPLOAD_TOKEN }}` with user `edx` didn't work for `repo-tools`, I've asked the question in #travis-to-github-actions-migration and asked SRE to help to resolve the issue.

Reply:
maybe user has to be `__token__` instead